src/userList2.js

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import UserList2ClientBreakdown from "./userList2ClientBreakdown";
+
+export default function UserList2(props) {
+
+    const userDataList = props.userDataList;
+
+    // create a distinct list of clients
+    let clientList = [];
+    userDataList.map(userData => {
+        clientList.push(userData.client)
+    });
+    clientList = clientList.filter((value, index, self) => {
+        return self.indexOf(value) === index;
+    })
```

Comment:
This can be simplified a bit
```suggestion
  // create a distinct list of clients
  const clientList = userDataList
    .map((user) => user.client)
    .filter((value, index, self) => self.indexOf(value) === index);
```